src/directives/vue-ng.js

```diff
@@ -68,10 +68,30 @@ function mounted (el, binding, vnode) {
       template.setAttribute(attrName, `${scopeName}($event)`);
     });
 
+    // for (const node of el.childNodes) {
+    //   el.removeChild(node);
+    //   template.appendChild(node);
+    // }
+
     const $compile = $injector.get('$compile');
     const bindFn = $compile(template);
     const [$ngElement] = bindFn($scope); // Bind to scope
 
+    if (transclude) {
+      // NOT THE IDEAL SOLUTION BU IT WORKS FOR BASIC ng-transclude
+      // https://docs.angularjs.org/api/ng/directive/ngTransclude#basic-transclusion
+      // It would have been better to work with slots but bannot access them from directive
```

Comment:
```suggestion
      // NOT THE IDEAL SOLUTION BUT IT WORKS FOR BASIC ng-transclude
      // https://docs.angularjs.org/api/ng/directive/ngTransclude#basic-transclusion
      // It would have been better to work with slots but cannot access them from directive
```